ci/hack/rosa_hcp_setup.sh

```diff
@@ -28,36 +18,44 @@ Configure resources for ROSA HCP cluster on AWS
 Mandatory arguments:
     -r, --region
         AWS region name.
-    --prefix
-        Prefix for the cluster name.
-    -v, --version
-        Version of the ROSA HCP cluster.
+    --name
+        Name for the cluster name.
     
 Optional arguments:
     -d, --debug
         Activate tracing/debug mode.
     -h, --help
         Display this message.
 Example:
-    ${0##*/} ./rosa_hcp_setup.sh --prefix test0612 -r us-east-1 -v 4.12
+    ${0##*/} ./rosa_hcp_setup.sh --name test0612 -r us-east-1
```

Comment:
What's the mean of `0612`? Is it just a random string?

Reply:
fixed.